crates/config/src/rule_config.rs

```diff
@@ -67,8 +67,18 @@ pub struct SerializableRuleConfig<L: Language> {
 }
 
 impl<L: Language> SerializableRuleConfig<L> {
+  fn get_default_message(&self) -> &str {
+    &self.id
+  }
+
   fn get_message(&self, node_match: &NodeMatch<StrDoc<L>>) -> String {
-    let bytes = self.message.generate_replacement(node_match);
+    // Note: The LSP client in vscode won't show any diagnostics at all if it receives one with an empty message
```

Comment:
hmm, this is not 100% correct since the empty message will be used in `sg scan` CLI. 
Can we move this to LSP?

Reply:
Okay I've submitted a change where it's only the diagnostic that has an empty message replaced with a default message. The message field is also used to generate the codeaction title, but vscode is happy to display a codeaction with an empty title, so I'm happy to leave the codeaction unchanged.